### ISSUE TITLE
Using BoundingBoxHelper

### ISSUE THREAD

**Issue by AlexRU18** [yandex/mapkit-android-demo#123]
CameraPosition constructor not contain method with BoundingBox/Helper.

How i can use Bounds with CameraPosition for autozoom by 2+ points?

**Comment by eberkovich** [comment]
You can use:
```
/**
 * Calculates the camera position that projects the specified area into
 * the view.
 */
 @NonNull
  public CameraPosition cameraPosition(
        @NonNull BoundingBox boundingBox);
```

**Comment by AlexRU18** [comment]
@eberkovich but this constructor is not resolved in last 3.4.0 version in Android
![2222](https://user-images.githubusercontent.com/16851339/64091706-71212f00-cd62-11e9-9a81-66dd8675139c.png)

**Comment by eberkovich** [comment]
It is method of the Map.
